tests/widget/inline/widget-inline.register.test.js

```diff
@@ -6,7 +6,7 @@ import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
 test('Inline widget with user registration', async ({ page }) => {
   const { clickButton, navigate } = asyncEvents(page);
 
-  await navigate('widget/inline?journey=Registration');
+  await navigate('widget/inline?journey=TEST_Registration');
```

Comment:
I take it that TEST_ is the prefix for all of the journeys we are reserving for automated tests from now on?

Reply:
That is correct. I accidentally broke some tests because I was altering a journey that Ryan was using for e2e, so I figured it would be better to have dedicated journeys for testing.